.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+on: [push]
+
+jobs:
+  build_image:
+    runs-on: ubuntu-latest
+    name: "Build and push image"
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Log in to ghcr.io
+      run: "echo ${{ secrets.RACKLET_CONTAINERS_GHCR }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin"
+    - name: Download tags, for use in the Makefile
```

Comment:
Instead of doing this; I should be able to just do a unshallow tag pull manually